augur/tasks/util/collection_util.py

```diff
@@ -477,29 +651,15 @@ def start_repos_from_given_group_of_users(session,limit,users,condition_string,p
     session.logger.info(f"valid repo git list: {tuple(valid_repo_git_list)}")
     
     #start repos for new primary collection hook
-    collection_size = start_block_of_repos(
-        session.logger, session,
-        valid_repo_git_list,
-        phases, repos_type=repos_type, hook=hook
-    )
-
-    return collection_size
-
-"""
-    Generalized function for starting a phase of tasks for a given collection hook with options to add restrictive conditions
-"""
-def start_repos_by_user(session, max_repo,phase_list, days_until_collect_again = 1, hook="core",new_status=CollectionState.PENDING.value,additional_conditions=None):
-
-    #getattr(CollectionStatus,f"{hook}_status" ) represents the status of the given hook
-    #Get the count of repos that are currently running this collection hook
-    status_column = f"{hook}_status"
-    active_repo_count = len(session.query(CollectionStatus).filter(getattr(CollectionStatus,status_column ) == CollectionState.COLLECTING.value).all())
-
-    #Will always disallow errored repos and repos that are already collecting
+    #collection_size = start_block_of_repos(
+    #    session.logger, session,
+    #    valid_repo_git_list,
+    #    phases, repos_type=repos_type, hook=hook
+    #)
 
-    #The maximum amount of repos to schedule is affected by the existing repos running tasks
-    limit = max_repo-active_repo_count
+    return valid_repo_git_list
 
+def split_random_users_list(session,status_col, status_new):
     #Split all users that have new repos into four lists and randomize order
```

Comment:
I don't see where this function is splitting the user list into 4 sections

Reply:
Oh you're totally right this will be fixed